src/treemap/inherent.rs

```diff
@@ -41,6 +41,31 @@ impl RoaringTreemap {
             .insert(lo)
     }
 
+    /// Adds a value to the set.
+    /// The value **must** be strictly bigger than the maximum value in the set.
```

Comment:
```suggestion
    /// The value **must** be greater or equal to the maximum value in the set.
```

Am I right on this? Can values be equal to the max even for treemaps?